.tekton/marin3r-operator-image-push.yaml

```diff
@@ -26,6 +36,12 @@ spec:
     value: quay.io/redhat-user-workloads/api-management-tenant/marin3r-operator-image:{{revision}}
   - name: dockerfile
     value: Containerfile.marin3r-operator
+  - name: prefetch-input
+    value: '{"type": "gomod", "path": "cloud-resource-operator/"}'
```

Comment:
it should point to marin3r-operator submodule here, and possibly to the api folder go.mod as well?

Reply:
updated, sorry. Thank you